src/workerd/io/worker-interface.capnp

```diff
@@ -245,9 +245,10 @@ interface EventDispatcher @0xf20697475ec1752d {
   # the outcome and whether the run should be retried. This does not complete immediately.
 
 
-  runAlarm @4 (scheduledTime :Int64) -> (result :AlarmRun);
+  runAlarm @4 (scheduledTime :Int64, retryNumber :UInt32) -> (result :AlarmRun);
   # Runs a worker's alarm.
   # scheduledTime is a unix timestamp in milliseconds for when the alarm should be run
+  # isRetry indicates whether this runAlarm is a retry or not.
```

Comment:
This comment needs to be updated to refer to `retryNumber` rather than `isRetry`